supybot_fedora/plugin.py

```diff
@@ -1202,7 +1263,7 @@ def quote(self, irc, msg, args, arguments):
         del symbols['UNH']
         del symbols['ANN']  # And this one is unused...
 
-        key_fmt = lambda d: ', '.join(sorted(d.keys()))
+        key_fmt = lambda d: ', '.join(sorted(d.keys()))  # noqa: E731
```

Comment:
We're defining quite a bit of lambda functions like this, we could move all that to a separate module (and `cmp` too), or properly define them with a `def` at the top of the file, no?

Reply:
yeah, i wanted to kind of leave this level of restructuring out of this PR, but my intent was to fix these afterwards.

filed the following ticket so i don't forget:
https://github.com/fedora-infra/supybot-fedora/issues/67